pycoolmasternet_async/coolmasternet.py

```diff
@@ -36,10 +37,19 @@ async def _make_request(self, request):
             reader, writer = await asyncio.open_connection(self._host, self._port)
 
             try:
-                writer.write(("\n").encode("ascii"))
-                prompt = await asyncio.wait_for(reader.readuntil(b">"), self._read_timeout)
-                if prompt != b"\r\n>":
-                    raise ConnectionError("CoolMasterNet prompt not found")
+                if self._consume_initial_prompt:
```

Comment:
@ashalita , can you elaborate on the logic here? IIUC, there is a new TCP connection for each request. The bootstrap logic is wrapped with `self._consume_initial_prompt` which means it will be executed only for the 1st request (1st command to CoolMaster). Is it intentional? What about the rest of the requests?